### PR TITLE
Feature/create default configuration

### PR DESCRIPTION
**Pull request by NoeliaBentancor**
## Description:

Add default creation of configuration.

## Related Issue (if appropriate):

https://app.asana.com/0/1205530699145161/1205952917063607

## Steps to test change/Steps to reproduce bug:

db db_video_hub should be empty

Run the following commands : 

1. docker exec -it video-hub bash

2. python manage.py migrate

### CODE REVIEW

File: web/src/components/OperatorSite/navigation.js [hunk 1]
```diff
@@ -47,6 +47,16 @@ export class OperatorRoutes extends Component {
         <Route exact path='/drop/:video-group_id/' component={VideoUploadPage} />
         <Route exact path='/drop/' component={VideoUploadPage} />
 
+        {/* <Route exact path='/fast-channels/' render={(props) => <FASTChannelDashboard history={history}
```
Comment (santiagomintegui) [inline web/src/components/OperatorSite/navigation.js:50]: why we have this code comment?

Comment (santiagomintegui) [inline web/src/components/OperatorSite/navigation.js]: why this code is comment?

Comment (NoeliaBentancor) [reply web/src/components/OperatorSite/navigation.js]: Solved, this should not be commited

File: hub_auth/migrations/default_23.py [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+
+
+        
+    def create_global_settings(apps, schema_editor):
+        from configuration.models import Configuration as configuration_model
+        config = configuration_model()
+        
+        
+        file_path = os.path.join(Migration.configuration_folder,'cloud_front_configuration.json')
+        print(file_path)
```
Comment (fpyoung) [inline hub_auth/migrations/default_23.py]: Remove logs please

File: hub_auth/migrations/default_23.py [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+    aws_default_region = os.environ.get('AWS_DEFAULT_REGION')
+    configuration_folder = os.path.join(os.path.dirname(__file__), 'configuration.samples/')
+    default_user = os.getenv('DEFAULT_USER')
+
+    
+    def assign_user_to_organization(apps, schema_editor,organization_name):
+        User = apps.get_model('hub_auth', 'account')
+        Organization = apps.get_model('organization', 'organization')
+        user = User.objects.get(
+            first_name ='noe'
```
Comment (fpyoung) [inline hub_auth/migrations/default_23.py]: Is this correct ?

File: hub_auth/migrations/default_23.py [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+        
+        from organization.models import Organization
+        import math
+        import time
+        new_organization = Organization()
+        new_organization.name = f'Default Organization{math.floor(time.time())}'
+        new_organization.plan_id = Migration.create_plan(apps, schema_editor)
+        new_organization.aws_account_id = aws_account_id
+        new_organization.save()
+        # Migration.assign_user_to_organization(apps, schema_editor,new_organization.name)
```
Comment (fpyoung) [inline hub_auth/migrations/default_23.py]: Maybe we should remove this now

Comment (NoeliaBentancor) [reply hub_auth/migrations/default_23.py]: Solved

**Review by santiagomintegui — APPROVED** [review]

**Review by fpyoung — APPROVED** [review]

**Pull request merged**